env.sqlite

```diff
@@ -2,3 +2,5 @@ PUID=1000
 PGID=1000
 PHP_TZ=UTC
 DB_CONNECTION=sqlite
+CACHE_DRIVER=file
+LOG_VIEWER_CACHE_DRIVER=file
```

Comment:
Pretty sure these don't belong here. These are just for testing and are Docker env, not Lychee.